src/main/java/Task.java

```diff
@@ -0,0 +1,14 @@
+public class Task {
+
+    public String taskName;
+    public String status;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        status = " ";
+    }
+
+    public void setStatusChar(String statusChar) {
```

Comment:
Maybe you should not name your String variable statusChar to avoid confusion with the Character class.